src/platform/site-wide/sass/style.scss

```diff
@@ -1,7 +1,6 @@
 $formation-image-path: "~@department-of-veterans-affairs/formation/assets/img";
 @import "modules/m-overrides";
 
-@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```

Comment:
How can we be sure `~@department-of-veterans-affairs/css-library/dist/tokens/scss/variables` isn't needed here?

Reply:
tldr: shared-variables was exposing a ton of what was already being imported through core.scss

I believe if it were needed here, the build would break on yarn watch or yarn build for undefined variables. 

The inheritance here is really convoluted, but a lot of what the modules in style.scss rely on are already being brought in from the core import below on line 4. 

For example, the crisis-line module uses the `media` mixin [here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/site-wide/sass/modules/_m-crisis-line.scss#L6). 

And even though the shared-variables file provided that mixin, so does the formation [core.scss file](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/core.scss#L21), through `base/b-breakpoints` import (don't ask me why that media mixin is in the breakpoints file instead of the formation mixin file, I really have no clue). 

Here's a screenshot from the review instance that shows the correct styles are being rendered: 
![image](https://github.com/user-attachments/assets/df0f5c68-8c15-4774-adf1-e50a837ee874)

The styles in the screenshot correspond to the `@include media` mixin in the crisis line module linked to above.

Reply:
Great, thanks for clearing this up @micahchiang!